bin/mapbox-gl-style-build

```diff
@@ -68,6 +79,17 @@ const buildStyles = async (styleDir, layerDir, outputDir, options = {}) => {
       process.exit(1);
     }
   });
+
+  const neverUsedVars = [...unusedContext].filter(
+    path => !usedContext.has(path)
+  );
```

Comment:
Do we need to make a copy of `unusedContext` here? `filter()` shouldn't modify the original so it seems cleaner to just

```suggestion
  const neverUsedVars = unusedContext.filter(
    path => !usedContext.has(path)
  );
```

Or am I missing something?

Reply:
Ah, this isn't about making a copy of it, it's about turning a set into an array with the spread operator. We're using a set so we don't have to dedupe, but ultimately we want an array.

Reply:
Ah okay I missed that.